firebase.json

```diff
@@ -568,6 +568,118 @@
         "source": "/avasummit",
         "destination": "/docs/vrf/v2/introduction/",
         "type": 301
+      },
+
+      /* Data feeds reorg */
+      {
+        "source": "/docs/using-chainlink-reference-contracts/",
```

Comment:
should we also add redirects for  `/docs/using-chainlink-reference-contracts` ?

Reply:
Both `/docs/using-chainlink-reference-contracts/` and `/docs/get-the-latest-price/` have redirects. What am I missing?

Reply:
Correct. This is complete.